conf/app.src

```diff
@@ -0,0 +1,6 @@
+SOURCE_URL=https://github.com/duniter/cesium/releases/download/v1.3.11/cesium-v1.3.11-web.zip
+SOURCE_SUM=
+SOURCE_SUM_PRG=
```

Comment:
```suggestion
SOURCE_SUM_PRG=sha256sum
```

Reply:
I deleted it because I didn't know how to complete the SOURCE_SUM= value.